astronomer/providers/amazon/aws/example_dags/example_create_emr_on_eks_cluster.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# create cluster
+eksctl create cluster \
+    --name $EKS_CONTAINER_PROVIDER_CLUSTER_NAME \
+    --region us-east-2 \
```

Comment:
region should come from an environment variable

Reply:
created env variables `INSTANCE_TYPE` and  `AWS_DEFAULT_REGION`